pegasus/sites.v3/code.org/public/teacher-dashboard/section_assessments.haml

```diff
@@ -4,188 +4,190 @@ theme: none
-      %td
-        {{assessment.status}}
-      %td
-        {{assessment.multi_correct}}
-      %td
-        {{assessment.multi_count}}
-      %td
-        {{assessment.multi_correct_percent}}
-      %td
-        {{assessment.timestamp}}
-
-:ruby
-  levels_filtered = "assessmentLevels | orderBy:orderAnswers | filter:(stageFilterAnswers && {stage: stageFilterAnswers}):true"
-
-%h1{'ng-show' => 'assessments.length'}= I18n.t('dashboard_assessments_answers')
-
-%div{style: "padding-left:20px", 'ng-show' => 'assessments.length'}
-  %table{style: "width:950px; table-layout:fixed;", 'ng-show' => 'assessments.length', "ng-form" => "allForm"}
-    %colgroup
-      %col{width: "100"}/
-      %col{width: "175"}/
-      %col{width: "75"}/
-      %col{width: "75"}/
-      %col{width: "275"}/
-      %col{width: "125"}/
-
-    %tr
-      %th.manage-th{colspan: 6}
-        %div{'ng-show' => 'assessmentStages && assessmentStages.length > 1', style: 'float: left; line-height: 35px'}
-          = I18n.t('dashboard_filter_by_stage')
-          %select{'ng-model' => 'stageFilterAnswers', "ng-options" => "stage for stage in assessmentStages", "ng-show" => 'assessmentStages'}
-            %option{value: ""}= I18n.t('dashboard_filter_all')
-        %button.btn.btn-white{'ng-csv' => levels_filtered,
-                filename: "assessment_levels.csv",
-                'csv-header' => "['Name', 'Stage', 'Puzzle', 'Question', 'Response', 'Correctness']",
-                'csv-column-order' => "['student.name', 'stage', 'puzzle', 'question', 'studentResult', 'correct']",
-                style: 'float: right'}
-          %i.fa.fa-download
-          = I18n.t('dashboard_download_csv')
-
-    %tr
-      %th.manage-th
-        %a{href: "", "ng-click" => "orderAnswers = 'student.name'"}= I18n.t('dashboard_students_name')
-      %th.manage-th
-        %a{href: "", "ng-click" => "orderAnswers = 'stage'"}= I18n.t('dashboard_stage')
-      %th.manage-th
-        %a{href: "", "ng-click" => "orderAnswers = 'puzzle'"}= I18n.t('dashboard_puzzle')
-      %th.manage-th
-        %a{href: "", "ng-click" => "orderAnswers = 'question'"}= I18n.t('dashboard_question')
-      %th.manage-th
-        %a{href: "", "ng-click" => "orderAnswers = ['!studentResult', 'studentResult']"}= I18n.t('dashboard_response')
-      %th.manage-th
-        %a{href: "", "ng-click" => "orderAnswers = 'correct'"}= I18n.t('dashboard_correctness')
-
-    %tr{"ng-repeat" => "level in #{levels_filtered}", "ng-form" => "form"}
-      %td
-        %a{"ng-href" => "#/sections/{{section.id}}/student/{{level.student.id}}"} {{level.student.name}}
-      %td
-        {{level.stage}}
-      %td
-        {{level.puzzle}}
-      %td
-        {{level.question}}
-      %td
-        {{level.studentResult}}
-      %td
-        {{level.correct}}
+#section-assessments-react{'ng-init' => '$emit("section-assessments-rendered");', 'ng-show' => 'react_assessments'}
+#section-assessments-angular{'ng-hide' => 'react_assessments'}
```

Comment:
The majority of this file is indenting everything below this line so that it gets hidden when the experiment is on.